website/views/static/press-kit.jade

```diff
@@ -19,7 +19,32 @@ block content
   p= env.t('presskitDownload')
     | &nbsp;
     a.btn.btn-success(href='/presskit/presskit.zip') presskit.zip
-  - var imgs = ['Habitica Text Logo', 'Habitica Logo Promo', 'Habitica Gryphon', 'Dread Drag\'on of Dilatory', 'Market Sample Screen', 'Challenges Sample Screen', 'Equipment Sample Screen', 'Guilds Sample Screen', 'HabiticaPromoThin', 'Laundromancer', 'Tasks Page', 'SnackLess Monster', 'Battling the Ghost Stag', 'Stagnant Dishes Boss', 'Necro-Vice']
+  - var imgs = ['Habitica Promo',
```

Comment:
That SO question should be correct. You need the dash on a line by itself and the variable declaration declared one indent in from that. I believe it's like this:

```jade
-
  var images = ['Habitica Promo',
    'Habitica Promo - Thin',
     // etc
```